src/commands/create/index.js

```diff
@@ -152,3 +152,20 @@ function listTypesInDirectory(directoryPath, types) {
     }
   }
 }
+
+function updateTemplateProperty(template, property, value) {
+  // If property in the template is missing, make sure we create the property and the value to it
+  if (!template[property]) {
+    template[property] = value;
+  } 
+  // If property in the template is an array
+  else if (Array.isArray(template[property])) {
+    template[property].push(value);
+  }
+  // If property is not an array we should convert it to an array and push the value
+  else if (!Array.isArray(template[property])) {
+    const existingValue = template[property];
+    template[property] = [existingValue, value];
```

Comment:
This will turn

```
Transform:
   - 'AWS::Serverless-2016-10-31'
```
into 
```
Transform:
   - 'AWS::Serverless-2016-10-31'
   - 'AWS::LanguageExtensions'
```

The original code added `AWS::LanguageExtensions` before the SAM transform. Not doing that could lead to issues

Reply:
Thanks. I noticed that but couldn't figure out why really. Anyway, a fix has been pushed 😃